arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -147,6 +152,14 @@ void BenchFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
                      "use waitForSync for created collections",
                      new BooleanParameter(&_waitForSync));
 
+  options->addOption("--create-database",
+                     "whether we should create the database specified via the server connection",
+                     new BooleanParameter(&_createDatabase));
+
+  options->addOption("--duration",
+                     "test for duration seconds instead of a fixed test count",
+                     new UInt64Parameter(&_duration));
```

Comment:
We normally also allow fractional values for seconds, at least in many other options. So we should maybe change this to a double parameter type?

Reply:
hm, the usecase here is rather to go into the range of hours, than fractions of seconds. 
Since current requests may need to finish first, it won't be completely "on time" anyways, so fragments would pretend a precision that can't be met anyways.